amr-wind/physics/multiphase/DamBreakFieldInit.cpp

```diff
@@ -35,6 +35,7 @@ void DamBreakFieldInit::operator()(
         const amrex::Real z = problo[2] + (k + 0.5) * dx[2];
 
         velocity(i, j, k, 0) = 0.0;
+        velocity(i, j, k, 1) = 0.0;
         velocity(i, j, k, 2) = 0.0;
```

Comment:
Why not just `velocity.setVal(0.0, 0, AMEX_SPACEDIM);`

Reply:
Sure !